index.html

```diff
@@ -5,8 +5,44 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Our Caterpiller</title>
+    <link rel="stylesheet" href="src/main.css">
 </head>
+
 <body>
-    
+    <header>
+        <h1>Hungry Caterpiller</h1>
+    </header>
+
+    <section id ="fruit-buttons">
+        <button class="fruit" value="apple">
+            <img src="assets/fruits/apple.png">
+        </button>
+        <button class="fruit" value="banana">
+            <img src="assets/fruits/banana.png">
+        </button>
+        <button class="fruit" value="blueberry">
+            <img src="assets/fruits/blueberry.png">
+        </button>
+        <button class="fruit" value="cherry">
+            <img src="assets/fruits/cherry.png">
+        </button>
+        <button class="fruit" value="green-apple">
+            <img src="assets/fruits/green-apple.png">
+        </button>
+        <button class="fruit" value="mango">
+            <img src="assets/fruits/mango.png">
+        </button>
+        <button class="fruit" value="orange">
+            <img src="assets/fruits/orange.png">
+        </button>
+    </section>
```

Comment:
Don't be afraid to add some white space between sections or other clearly defined parts of the code both in HTML and JS. 

Adding some but not too much white space makes the code easier to read at a moments glance.